pandas/tests/io/parser/test_parsers.py

```diff
@@ -99,3 +102,14 @@ def read_table(self, *args, **kwds):
         kwds = kwds.copy()
         kwds['engine'] = self.engine
         return read_table(*args, **kwds)
+
+
+class TestParameterValidation(object):
```

Comment:
Yikes!  You're right.  We changed our minds about that.  Mind fixing the documentation on that in a separate commit / PR?